### PR TITLE
📝🍎🚫💰 Update Swagger API documentation to explain non-revenue service representation and prediction 'uncertainty'

### PR DESCRIPTION
**Pull request by bfauble**
#### Summary of changes

**Asana Ticket:** [📝🍎🚫💰 Update Swagger API documentation to explain non-revenue service representation and prediction 'uncertainty'](https://app.asana.com/0/584764604969369/1205761861495469/f)

* added `Prediction` docs for arrival/departure uncertainty values
* added `revenue` filter docs for `Trip`, `Prediction`, and `Vehicle`.

Expected Merge Dec 7

### CODE REVIEW

File: apps/api_web/lib/api_web/controllers/prediction_controller.ex [hunk 1]
```diff
@@ -261,6 +261,61 @@ defmodule ApiWeb.PredictionController do
...
+
+              | Value            | Description |
+              |------------------|-------------|
+              | < 300 or omitted |	Valid real-time prediction |
+              | 300              |  Real-time prediction not available. This code is primarily used when a vehicle has not yet been assigned to the trip, (i.e. because the block has not started yet). It is a schedule-based prediction, but Swiftly adjusts the schedule-based prediction time using observed historical travel times to make predictions more accurate than the schedule |
+              | 301              |	Valid real-time prediction, though the bus appears to be stalled or significantly delayed and predictions are not as accurate |
+              | > 301            |	Likely invalid prediction, recommend not showing anything (and not showing scheduled time), very rare situation |
+
+
+              Subway
```
Comment (fsaid90) [inline apps/api_web/lib/api_web/controllers/prediction_controller.ex:281]: @bfauble we might want to say Subway/Light Rail

File: apps/api_web/lib/api_web/controllers/prediction_controller.ex [hunk 2]
```diff
@@ -261,6 +261,61 @@ defmodule ApiWeb.PredictionController do
...
+              See [entities tripUpdate stop_time_updates departure uncertainty](https://swiftly-inc.stoplight.io/docs/realtime-standalone/613d1d7f1eae3-gtfs-rt-trip-updates)
+
+              | Value            | Description |
+              |------------------|-------------|
+              | < 300 or omitted |	Valid real-time prediction |
+              | 300              | Real-time prediction not available. This code is primarily used when a vehicle has not yet been assigned to the trip, (i.e. because the block has not started yet). It is a schedule-based prediction, but Swiftly adjusts the schedule-based prediction time using observed historical travel times to make predictions more accurate than the schedule |
+              | 301              |	Valid real-time prediction, though the bus appears to be stalled or significantly delayed and predictions are not as accurate |
+              | > 301            |	Likely invalid prediction, recommend not showing anything (and not showing scheduled time), very rare situation |
+
+              Subway
```
Comment (fsaid90) [inline apps/api_web/lib/api_web/controllers/prediction_controller.ex:308]: Same here as above

File: apps/api_web/lib/api_web/swagger_helpers.ex [hunk 3]
```diff
@@ -273,6 +273,8 @@ defmodule ApiWeb.SwaggerHelpers do
       """
       #{opts[:desc]}
 
+      Revenue status indicates whether or not the vehicle is accepting passengers.
```
Comment (fsaid90) [inline apps/api_web/lib/api_web/swagger_helpers.ex:276]: Suggested reword:

"Revenue status indicates whether or not the vehicle is in revenue service - i.e. whether or not a vehicle is accepting passengers."

**Review by shantigonzales — CHANGES_REQUESTED** [review]
Reviewing now, seems like our name change hasn't made it in, but other than that it's looking good to me!

**Review by Whoops — APPROVED** [review]

**Review by shantigonzales — APPROVED** [review]

**Pull request merged**